### ISSUE TITLE
Cannot change the arm and leg positions using a keyboard with iOS

### ISSUE THREAD

**Issue by jobara** [phetsims/john-travoltage#118]
Opened the [John Travoltage](http://www.colorado.edu/physics/phet/dev/html/john-travoltage/1.3.0-accessible-instance.1/john-travoltage_en.html?accessibility) sim in Safari ( iOS 9.3.5 ) and attempted to interact with the sim via a connected bluetooth keyboard (apple keyboard). 

I was able to put focus on the hand and foot with the keyboard, but not able to change their position using the arrow keys. I attempted to do this with VoiceOver enabled and was able to move the foot only. I attempted to move focus to the hand, but while the sim focus would move, VoiceOver's focus seemed to stay on the foot. ( VoiceOver's black outline remained fixed on the same parallel DOM element ).

This may be related to https://github.com/phetsims/balloons-and-static-electricity/issues/164

**Comment by jobara** [comment]
@jessegreenberg i tried this in iOS 10 and had the same issue.

**Comment by jobara** [comment]
@jessegreenberg I tried with a non-apple bluetooth keyboard and experienced the same problem.

**Comment by jessegreenberg** [comment]
Interesting, thanks @jobara.  Does the problem still happen when using the bluetooth keyboard with a non-iOS device?

**Comment by jobara** [comment]
@jessegreenberg i tried on macOS and didn't have the issue there.

**Comment by jobara** [comment]
I've tried this again using a plain HTML5 slider and couldn't get it to function either with the bluetooth keyboard.

**Comment by jobara** [comment]
Tried on iOS with a USB keyboard through the camera connection kit and had the same keyboard interaction issue as stated above.

**Comment by jessegreenberg** [comment]
@jobara and I create a JSFiddle to track this issue in another context.

https://jsfiddle.net/25ejnztt/1/

**Comment by jessegreenberg** [comment]
This issue is going to be tracked in phetsims/a11y-research#10.
